src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -607,6 +612,7 @@
 					} else {
 						if (itemElement.parentNode !== null && item.current.scale < 0.01) {
 							itemElement.parentNode.removeChild(itemElement);
```

Comment:
You don't need removeChild when in next line you are appending this element to new parent.

Reply:
Right. Thanks for suggestion.